src/Bicep.LangServer/Bicep.LangServer.csproj

```diff
@@ -11,11 +11,18 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json.Schema" Version="3.0.14" />
```

Comment:
This library has some strange commercial licensing setup attached to it which I've never fully understood. Are there any issues using it here? See https://www.newtonsoft.com/jsonschema for more info.

Reply:
At one point it’d be nice to have errors on invalid values, but viscose can provide that out of the box, I think we don’t need to build validation in bicep itself.

Reply:
Thanks for pointing this. Looks like it was pulled out of Json.Net(which is under MIT license) and is not free anymore.

I am trying couple other libraries. Let me know if you have any suggestions.

Reply:
I think I can use this one instead: https://www.nuget.org/packages/NJsonSchema/. It's under MIT license.

Reply:
As discussed offline, removed the validation as part of this commit:
https://github.com/Azure/bicep/commit/7c53107f3fb09a3e6403cf76b8ee6a876c13806c